docs/components/rpa/getting-started.md

```diff
@@ -65,7 +68,7 @@ Once you are happy with your script and have tested it locally, you can start au
    1. Deploy the BPMN model with the configured RPA task.
```

Comment:
>   1. Deploy the BPMN model with the configured RPA task.

We should probably note how to deploy here.

Reply:
The pattern is the same as the RPA file (🚀 icon). We can align the wording

```suggestion
   1. Deploy the BPMN model with the configured RPA task by clicking on the rocket (🚀) icon in Modeler.
```